pkg/plan/recipe/install_plans.go

```diff
@@ -310,14 +310,16 @@ func BuildK8SPlan(kubernetesVersion string, kubeletNodeIP string, seLinuxInstall
 func BuildCNIPlan(cni string, manifests [][]byte) plan.Resource {
 	b := plan.NewBuilder()
 
-	if len(manifests) != 1 {
-		log.Fatalf("install-cni: expected 1 manifest for the CNI plugin, got %d", len(manifests))
-	}
-
 	b.AddResource(
 		"install-cni:apply-manifests",
 		&resource.KubectlApply{Manifest: manifests[0], Filename: object.String(cni + ".yaml")},
 	)
+	if len(manifests) == 2 {
+		b.AddResource(
+			"install-cni:apply-manifests-ds",
+			&resource.KubectlApply{Manifest: manifests[1], Filename: object.String(cni + "-daemon-set" + ".yaml")},
```

Comment:
It would be good to make this resource depend on the install-cni:apply-manifests resource

Reply:
Good point, added the dependency